cmd/antrea-agent/agent.go

```diff
@@ -864,6 +865,14 @@ func run(o *Options) error {
 			return fmt.Errorf("failed to connect uplink to OVS bridge: %w", err)
 		}
 	}
+	// secondaryNetworkController Initialize must be run after FlowRestoreComplete.
+	if features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) {
+		defer secondaryNetworkController.RestoreHostInterfaceConfiguration()
```

Comment:
nit: maybe name it `secondaryNetworkController.Restore()` to hide details from the program entry and to be consistent with the other functions: `Initialize`, `Run`.

Reply:
Sure, updated.